src/inspector/internal/inspectorcontroller.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "inspectorcontroller.h"
+#include "inspector/models/popups/abstractelementpopupmodel.h"
+
+bool mu::inspector::InspectorController::supportsPopup(engraving::ElementType elementType) const
```

Comment:
Preferably add `using namespace mu::inspector;` (followed by a newline), and then remove `mu::inspector::` from the rest of the file